contrib/win32/win32compat/misc.c

```diff
@@ -314,22 +363,35 @@ spawn_child(char* cmd, int in, int out, int err, DWORD flags) {
 	si.dwFlags = STARTF_USESTDHANDLES;
 
 	b = CreateProcessW(NULL, cmd_utf16, NULL, NULL, TRUE, flags, NULL, NULL, &si, &pi);
+	GOTO_CLEANUP_IF(!b, ENOEXEC);
+	CloseHandle(pi.hThread);
```

Comment:
**CreateProcessW** - particular function, which return two handles. Both handles must be closed.